mujina-idp/saml-users.yml

```diff
@@ -1,7 +1,13 @@
 samlUserStore:
   samlUsers:
-    - username: test-user
-      password: test password
+    - username: James Conlon
+      password: welcome1
       samlAttributes:
-        attribute 1: test attribute 1 value
-        attribute 2: test attribute 2 value
+        LAA_ACCOUNTS: 12345
+        LAA_APP_ROLES: CCR_CCRAdministrator,CCR_CCRDirector
+    - username: ABRA-S
```

Comment:
This repository should not have any LAA-specific data.  Any seed data should be defined when the application is started up by specifying a custom .yml file in combination with the _spring.config.location_ parameter.

Reply:
I think we need a chat to clarify how we're going to use this.